packages/amplication-server/src/core/build/build.service.ts

```diff
@@ -384,6 +384,7 @@ export class BuildService {
             projectId: commitWithAccount.commit.projectId,
             buildId: buildId,
             externalId: encryptString(commitWithAccount.commit.user.id),
+            envBaseUrl: process.env.CLIENT_HOST,
```

Comment:
Please leverage the configService instead of reading environment variables as they might not be present in other environments.